plugins/org.jkiss.dbeaver.ext.mysql/plugin.xml

```diff
@@ -311,6 +311,7 @@
             <nativeClients>
                 <client id="mariadb_client" label="MariaDB Binaries">
                     <dist os="win32" targetPath="clients/mariadb/win" remotePath="repo:/drivers/mariadb/client/win" resourcePath="clients/mariadb/win">
+                        <driver id="mariaDB"/>
```

Comment:
Don't we need to add the same line for MySQL?

Reply:
No; it doesn't work the opposite way